packages/core/src/App/Components/Elements/login-prompt.jsx

```diff
@@ -36,6 +35,7 @@ LoginPrompt.propTypes = {
     page_title: PropTypes.string,
 };
 
+
```

Comment:
please remove the extra line, and return url for import on top

Reply:
extra line is removed. And now it's the exactly like before, but I'm moving LoginPrompt.defaultProps to top. That's what you asking, yeah? :)